### PR TITLE
Example

### PR DESCRIPTION
**Pull request by ldragoon**
Updated the file structure & cleanedup README file.

### CODE REVIEW

**Comment by ldragoon** [comment]
@qjhart - ready for you to look at. :)

File: Makefile [hunk 1]
```diff
@@ -1,48 +1,116 @@
 #! /usr/bin/make -f
+
 SHELL:=/bin/bash
 
+define pod
```
Comment (qjhart) [inline Makefile]: This is a method for creating documentation in tools like Makefiles, or  things that don't have other methods.

File: Makefile [hunk 2]
```diff
@@ -1,48 +1,116 @@
...
 
 trello=http https://api.trello.com/1/boards/${board}/ key==${key} token==${token}
 
-import:${board}/board.json
 
-files:=$(patsubst %,${board}/%.json,lists cards)
+.PHONY: INFO check
+
+INFO::
+	@pod2usage -exit 0 ${MAKEFILE_LIST}
```
Comment (qjhart) [inline Makefile]: `pod2usage` shows the synopsis, `pod2text` shows complete documentation.  Now if you type `make` you'll get some helper info.

File: Makefile [hunk 3]
```diff
@@ -1,48 +1,116 @@
...
+	- pY20Yz5x.json (This is the downloaded board)
+	- pY20Yz5z.ttl ( This is created from trello to moment )
+	- cats
+		- cats.json ( converted from ../pY20Yz5z.ttl with RIOT )
+			- Z4444 (a card)
+					- image_name.jpg
+
+=cut
+
+endef
```
Comment (qjhart) [inline Makefile]: This is a method for creating documentation in tools like Makefiles, or  things that don't have other methods.  You can look at https://perldoc.perl.org/perlpod.html for more information

File: Makefile [hunk 4]
```diff
@@ -1,48 +1,116 @@
...
-${board}/board.json:
-	[[ -d ${board} ]] || mkdir ${board};\
-	${trello} lists==all cards==all card_attachments==true | jq . > $@
+import:${board}.json thumbnails
 
-thumbnails: ${board}/board.json
-	[[ -d ${moment} ]] || mkdir ${moment};\
+${board}.json:
+	[[ -d ${moment} ]] || mkdir moments/${moment}; \
+	${trello} lists==all cards==all card_attachments==true | jq . > moments/${moment}/$@
```
Comment (qjhart) [inline Makefile]: No `moments` directory  (anywhere) 

```suggestion
	${trello} lists==all cards==all card_attachments==true | jq . > ${moment}/$@
```

File: Makefile [hunk 5]
```diff
@@ -1,48 +1,116 @@
...
-		echo https://api.trello.com/1/cards/$$a key==${key} token==${token} ;\
+		echo https://api.trello.com/1/cards/$$a key==${key} token==${token}; \
 	  b=$$(basename $$url); \
-	  [[ -d ${moment}/$$l ]] || mkdir ${moment}/$$l; \
-		[[ -f ${moment}/$$l/$$b ]] || http $$url > ${moment}/$$l/$$b ;\
-	  echo "${moment}/$$l/$$b";\
+	  [[ -d ${moment}/$$l ]] || mkdir moments/${moment}/${moment}/$$l; \
+		[[ -f ${moment}/$$l/$$b ]] || http $$url > moments/${moment}/${moment}/$$l/$$b; \
+	  echo "${moment}/$$l/$$b"; \
 	done
```
Comment (qjhart) [inline Makefile]: Remove `moments` subdir

File: Makefile [hunk 6]
```diff
@@ -1,48 +1,116 @@
...
+	${riot} --formatted=ttl --base=z: moments/${moment}/${moment}_t.ttl | sed -e 's/<z:/</g' > moments/${moment}/$@
+	rm -f moments/${moment}/${moment}_t.ttl
 
-${moment}.json:
-	${riot} --formatted=jsonld --base=z: ${moment}.ttl | sed 's/z:#//' > $@
\ No newline at end of file
+${moment}.json: moments/${moment}/${board}.ttl
+	[[ -d ${moment} ]] || mkdir moments/${moment}/${moment}; \
+	rm -f moments/${moment}/${moment}/${moment}.json; \
+	${riot} --formatted=jsonld --base=z: moments/${moment}/${board}.ttl | sed 's/z:#//' > moments/${moment}/${moment}/$@
```
Comment (qjhart) [inline Makefile]: rm `moments` sudir

File: Makefile [hunk 7]
```diff
@@ -1,48 +1,116 @@
...
-${moment}.ttl:${board}/board.json
-	./trello2moment --board=${board} 2>${moment}.err > ${moment}_t.ttl
-	${riot} --formatted=ttl --base=z: ${moment}_t.ttl  | sed -e 's/<z:/</g' > $@
-	rm -f ${moment}_t.ttl
+${board}.ttl: moments/${moment}/${board}.json
+	./trello2moment --moment=${moment} --board=${board} 2>moments/${moment}/${board}.err > moments/${moment}/${board}_t.ttl
+	${riot} --formatted=ttl --base=z: moments/${moment}/${board}_t.ttl | sed -e 's/<z:/</g' > moments/${moment}/$@
+	rm -f moments/${moment}/${board}_t.ttl
+
+${moment}_moment.ttl: moments/${moment}/${board}.json
```
Comment (qjhart) [inline Makefile]:
```suggestion
${moment}.ttl: ${moment}/${board}.json
 ```

File: README.md [hunk 8]
```diff
@@ -50,21 +50,33 @@ Run `echo $key && echo $token`.
...
+#### Create .JSON File
+
+Run `make key=$key token=$token board=BOARD_NAME moment=MOMENT_NAME [MOMENT_NAME].json`
 
 #### Trello2Moment
 
-Run `node trello2moment --board=BOARD_NAME`
+Run `node trello2moment --board=BOARD_NAME --moment=MOMENT_NAME [--description=true]`
 
 ## LDP
```
Comment (qjhart) [inline README.md]: Some of this info can go into the `POD` in the Makefile now.

File: Makefile (hunk 2 above)
Comment (ldragoon) [reply Makefile]: @qjhart - I ran the code and it works correctly. :)

File: Makefile (hunk 4 above)
Comment (ldragoon) [reply Makefile]: Removed all instances of moments dir.

File: README.md (hunk 8 above)
Comment (ldragoon) [reply README.md]: I simplified the ReadMe and I'm currently working on moving it into the POD now.

**Pull request merged**